models/category/src/category-draft/presets/sample-data-goodstore/bar-glassware.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import kitchen from './kitchen';
+
+const kitchenDraft = kitchen().build<TCategoryDraft>();
+
+const barGlassware = (): TCategoryDraftBuilder =>
```

Comment:
Can this please be updated to barAndGlassware?

Reply:
Sure can - updated with [9750fc5](https://github.com/commercetools/test-data/pull/357/commits/9750fc5413de175e44bda67acf2877f39af6c75b)

Reply:
Can we stick to the original names, otherwise the product exporter needs to know about this change, which is custom. Alternatively we would always need to update 150 files manually.

Reply:
It would be nice to stick to a consistent naming convention. This is the only category that fails to do so. Can the product exporter be updated?

Reply:
no worries, will do

Reply:
Keeping this preset name as `barGlassware` 👍